### PR TITLE
Updated FR translations

### PR DESCRIPTION
**Pull request by florentdordaquy**
Hello,

Here is all the missing translations.
Sorry for the delay 

Merci :)

### CODE REVIEW

File: messages/ui-fr.json [hunk 1]
```diff
@@ -633,29 +634,29 @@
...
-        "ResultCount": "Result Count",
+        "ResultCount": "Nombre Résultats",
         "RecipeMode": {
-          "Title": "Recipe Mode",
+          "Title": "Mode Recette",
           "Auto": "Auto",
-          "Manual": "Manual",
+          "Manual": "Manuel",
           "Message": {
-            "Auto": "Auto recipe mode ignores the given cooking schedule, but respects the recipe levels. It finds the best cooking schedule and the team using the given recipe levels instead. As a result, the calculation will take longer to complete."
```
Comment (RaenonX) [inline messages/ui-fr.json:642]: Shouldn't have extra space after `"Auto"`

File: messages/ui-fr.json [hunk 2]
```diff
@@ -604,7 +605,7 @@
           "FinalEvolution": "Évolution finale d'aperçu"
         },
         "Message": {
-          "RatingResultsStoredOnline": "Premium users can have their rating results persisted online for the same build to save repetitive computation. [Get Premium Now!](https://pks.raenonx.cc/en/docs/view/subscription/intro)"
```
Comment (RaenonX) [inline messages/ui-fr.json:607]: Does French separate punctuations (`!` here) with word like `maintenant !`, or it's `maintenant!`?

**Review by RaenonX — CHANGES_REQUESTED** [review]
Remember to just amend your last commit, so all the translations are in the same commit

File: messages/ui-fr.json (hunk 2 above)
Comment (florentdordaquy) [reply messages/ui-fr.json:607]: Yes, there is a space before and after the !, it's the same for : ; « » and also math symbols

**Comment by RaenonX** [comment]
@florentdordaquy Remember to push your change, in case you forgot :)

**Review by RaenonX — APPROVED** [review]

**Pull request merged**